tests/e2e/util.go

```diff
@@ -1426,13 +1513,13 @@ func verifyCNSFileAccessConfigCRDInSupervisor(ctx context.Context, f *framework.
 	}
 }
 
-//verifyEntityReferenceForKubEntities takes context,client, pv, pvc and pod as inputs
-//and verifies crdCNSVolumeMetadata is attached
-
-func verifyEntityReferenceForKubEntities(ctx context.Context, f *framework.Framework, client clientset.Interface, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim, pod *v1.Pod) {
+// verifyEntityReferenceForKubEntities takes context,client, pv, pvc and pod
+// as inputs and verifies crdCNSVolumeMetadata is attached.
+func verifyEntityReferenceForKubEntities(ctx context.Context, f *framework.Framework,
+	client clientset.Interface, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim, pod *v1.Pod) {
```

Comment:
Is this a white space ?

Reply:
Can you be more specific? I do not understand your comment.

Reply:
Line 1059 in the code-diff shows a highlighted green block at the beginning of the line, as it stands out from the other code diffs I am thinking it could be a user typed whitespaces.

Reply:
I didn't change L1059. I suppose you mean L1519 has a leading tab, which is needed for indentation.

BTW, "go fmt" automatically corrects indentations.